### ISSUE TITLE
Misspelled AIRA-INVALID attribute

### ISSUE THREAD

**Issue by pczarn2** [opena11y/ainspector-for-firefox#45]
Under the Forms -> Rule -> Information on invalid values -> Techniques, there is a misspelled attribute AIRA-INVALID, it should be ARIA-INVALID instead. https://raw.githubusercontent.com/opena11y/ainspector-for-firefox/main/src/content-script.js Line 42616

**Comment by jongund** [comment]
@pczarn2 
Thank you for reporting this issue, I will get an update to AInspector out soon.
